forms-web-app/src/config.js

```diff
@@ -3,6 +3,14 @@ module.exports = {
     timeout: Number(process.env.APPEALS_SERVICE_API_TIMEOUT || 10000),
     url: process.env.APPEALS_SERVICE_API_URL,
   },
+  fileUpload: {
```

Comment:
The reason I haven't done it centrally is because I was loosely aware of several other places uploads are allowed, and didn't know if we always allow the same file types every time. 

Unless I've misunderstood your point here.

Reply:
Good point. They _SHOULD_ be the same each time, but you're right not to lock it together.